src/main/config/README.txt

```diff
@@ -71,7 +71,7 @@ or
 LOGIN DETAILS (remember to change the password immediately after installation in a production environment)
 =============
 username - admin
-password - test
+password - Admin123
```

Comment:
Did you notice the different passwords that you need to use depending on which installation type you chose on the first screen?

Reply:
@shiddy did you see the above comment?